server/sonar-web/src/main/js/apps/tutorials/onboarding/Onboarding.js

```diff
@@ -72,28 +71,26 @@ export default class Onboarding extends React.PureComponent {
 
   componentDidMount() {
     this.mounted = true;
-    this.attachShortcuts();
+
+    // useCapture = true to receive the event before inputs
+    window.addEventListener('keydown', this.onKeyDown, true);
+
     if (!this.props.currentUser.isLoggedIn) {
       handleRequiredAuthentication();
     }
   }
 
   componentWillUnmount() {
+    window.removeEventListener('keydown', this.onKeyDown, true);
     this.mounted = false;
-    this.detachShortcuts();
   }
 
-  attachShortcuts() {
-    key.setScope('onboarding');
-    key('esc', 'onboarding', () => {
+  onKeyDown = (event /*: KeyboardEvent */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)